lib/active_support/cache/database_store.rb

```diff
@@ -126,17 +125,21 @@ def write_multi_entries(hash, **_options)
 
           {
             key: key,
-            value: Marshal.dump(entry.value),
-            version: entry.version.presence,
-            expires_at: expires_at,
             created_at: Time.zone.now,
+            **entry_attributes(entry),
           }
         end
 
         # In rails 7, we can use update_only not to do anything. But for the sakes of compatibility, we don't use any additional parameters.
         @model.upsert_all(entries)
       end
 
+      def entry_attributes(entry)
```

Comment:
should be private

Reply:
It is, all these methods are listed under "private" directive.